ssr/index.md

```diff
@@ -1,28 +1,25 @@
-# Aún no está traducido...
----
-
-# Server Side Rendering (SSR)
+# Renderizado del Lado del Servidor (SSR) {#server-side-rendering-ssr}
 
 :::tip
-If you are using **Nuxt.js,** you need to read [**these instructions**](./nuxt.md) instead.
+Si estás usando **Nuxt.js,** necesitas leer [**estas instrucciones**](./nuxt.md) en lugar de estas.
 :::
 
-Creating stores with Pinia should work out of the box for SSR as long as you call your `useStore()` functions at the top of `setup` functions, `getters` and `actions`:
+Crear almacenes con Pinia debería funcionar por defecto para SSR siempre y cuando llames a tus funciones `useStore()` en lo más lato de tus funciones `setup`, `getters` y `acciones`:
```

Comment:
I think that we can improve  this translate to: "La creación de almacenes con Pinia debería funcionar de forma inmediata para SSR siempre y cuando llame a sus funciones `useStore()` en la parte superior de las funciones `setup`, `getters` y `acciones`". Do you agree?

Reply:
Sure :)